README.md

```diff
@@ -31,167 +31,119 @@ Where `<snapshot version>` refers to the version in the Sonatype Snapshot badge
 
 ## Usage
 
-### Elastic Request
+In order to get the functional effect of executing a specified `Elasticsearch` request, we must provide the `Elasticsearch` layer to that effect.  We can then call the `execute`method defined in `Elasticsearch` that accepts `ElasticRequest` as parameter.
```

Comment:
I would rephrase this to something like:
```
In order to get the functional effect of executing a specified `Elasticsearch` request, we must provide the `Elasticsearch` layer to that effect.
```
In order to execute an Elasticsearch request, we can rely on the `Elasticsearch` layer which offers an `execute` method accepting an `ElasticRequest`. In order to build the `Elasticsearch` layer we need to provide the following layers: 
```